crawl-ref/source/state.h

```diff
@@ -30,6 +30,16 @@ struct god_act_state
     int      depth;
 };
 
+struct map_mode_info_t
+{
+public:
+    // If we are in map mode, absolute level pos of the cursor (position and branch id)
+    level_pos cursor_lpos;
+
+    // If we are in map mode, the level id we started map mode from (i.e. your starting branch+depth).
```

Comment:
"starting" is a weird word to describe this, but I don't see anything obviously better. "The level the player is actually on" seems okay but it leaves murky the question of "the player character" or "the attention of the human being playing the game".

I'm okay leaving as-is if you don't have any better ideas either.

Reply:
I think that "the level the player character is on" works as it's more accurate in the probably-impossible-but-maybe-someday case that we start map mode from somewhere that is not the player's position.

Reply:
(or specifically, the "player character" phrasing.)